cdap-app-templates/cdap-etl/cdap-etl-realtime/src/test/java/co/cask/cdap/template/etl/realtime/RealtimeElasticsearchSinkTest.java

```diff
@@ -138,7 +138,7 @@ public void testESSink() throws Exception {
       AdapterManager adapterManager = createAdapter(adapterId, adapterConfig);
 
       adapterManager.start();
-      TimeUnit.SECONDS.sleep(5);
+      TimeUnit.SECONDS.sleep(10);
```

Comment:
@bdmogal I think that works well only for Batch Adapters. For realtime adapters, the run doesn't complete until we stop the adapter?